settings.py

```diff
@@ -6,3 +6,11 @@
 
 SLACK_SIGNING_SECRET = os.environ["SLACK_SIGNING_SECRET"]
 SLACK_TOKEN = os.environ["SLACK_TOKEN"]
+
+
+# Feature toggles
+JIRA_USERNAME = os.environ["JIRA_USERNAME"]
+JIRA_PASSWORD = os.environ["JIRA_PASSWORD"]
+JIRA_SERVER = os.environ["JIRA_SERVER"]
+JIRA_PROJECT_KEY = os.environ["JIRA_PROJECT_KEY"]
+CREATE_JIRA_VERSIONS = os.environ["CREATE_JIRA_VERSIONS"]
```

Comment:
https://github.com/nhsd-a2si/dos-status-monitor/blob/develop/dos_status_monitor/config.py

I normally write myself a little helper function to get a bool out of an environment variable - see example on above link